Tools/CISupport/ews-build/events.py

```diff
@@ -315,26 +315,32 @@ def _get_commit_msg(self, repo, sha):
 
     @defer.inlineCallbacks
     def _get_pr_files(self, repo, number):
-        # Copied from https://github.com/buildbot/buildbot/blob/v2.10.5/master/buildbot/www/hooks/github.py to include added/modified/deleted
+        PER_PAGE = 100
+
         headers = {"User-Agent": "Buildbot"}
         if self._token:
             headers["Authorization"] = "token " + self._token
 
-        url = "/repos/{}/pulls/{}/files".format(repo, number)
-        http = yield httpclientservice.HTTPClientService.getService(
-            self.master,
-            self.github_api_endpoint,
-            headers=headers,
-            debug=self.debug,
-            verify=self.verify,
-        )
-        res = yield http.get(url)
-        if 200 <= res.code < 300:
-            data = yield res.json()
-            return [self.file_with_status_sign(f) for f in data]
-
-        log.msg('Failed fetching PR files: response code {}'.format(res.code))
-        return []
+        page = 1
```

Comment:
Is this variable needed? I don't see it being used anywhere.

Reply:
That means this PR isn't actually working, then....a moment to update and try and exercise the change locally

Reply:
Updated. We do need the `page` variable because if more than 100 files are changed, GitHub will force pagination.